yarn-project/aztec/terraform/node/variables.tf

```diff
@@ -64,27 +64,16 @@ variable "P2P_MAX_PEERS" {
 }
 
 variable "P2P_ENABLED" {
-  type = bool
+  type    = bool
+  default = true
 }
 
-variable "AVAILABILITY_ORACLE_CONTRACT_ADDRESS" { type = string }
-variable "ROLLUP_CONTRACT_ADDRESS" { type = string }
-variable "REGISTRY_CONTRACT_ADDRESS" { type = string }
-variable "INBOX_CONTRACT_ADDRESS" { type = string }
-variable "OUTBOX_CONTRACT_ADDRESS" { type = string }
-variable "GAS_TOKEN_CONTRACT_ADDRESS" { type = string }
-variable "GAS_PORTAL_CONTRACT_ADDRESS" { type = string }
-variable "AGENTS_PER_SEQUENCER" { type = string }
 variable "PROVING_ENABLED" {
   type    = bool
   default = true
 }
 
-variable "IMAGE_TAG" {
-  type = string
-}
-
-variable "FULL_IMAGE" {
+variable "AGENTS_PER_SEQUENCER" {
```

Comment:
I think this is only needed in the prover configuration, not the node/sequencer.